### PR TITLE
Refs [TASK] [36254] Tạo buttom navigation

### PR DESCRIPTION
**Pull request by hoangnguyen424331**
## Related Tickets
- [Refs [TASK] [36254] [Tạo buttom navigation]](https://edu-redmine.sun-asterisk.vn/issues/36254)
## WHAT
- Tạo buttom navigation
## Evidence (Screenshot or Video)
Screen Shot 2021-05-20 at 10 40 38 AM

## Review Checklist
https://edu-redmine.sun-asterisk.vn/issues/36254
Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/mFrameMain"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
    android:id="@+id/frameMain"
```

File: app/src/main/res/layout/fragment_home_page.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    android:textSize="@dimen/sp_20"
+                    android:textStyle="bold" />
+            </RelativeLayout>
+
+            <androidx.appcompat.widget.AppCompatEditText
+                android:id="@+id/edtSearch"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/dp_40"
+                android:layout_centerHorizontal="true"
+                android:layout_marginLeft="@dimen/dp_40"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_home_page.xml]: dùng cố định 1 style e.
Chổ thì dùng start chổ thì dùng left thế

Comment (hoangnguyen424331) [reply app/src/main/res/layout/fragment_home_page.xml]: dạ anh

File: app/src/main/java/com/sun/cooktails_app/ui/homepage/HomePageFragment.kt [hunk 3]
```diff
@@ -1,6 +1,55 @@
...
+        fragmentManager?.let {
+            viewPagerHomePage.adapter = HomePageAdapter(it, listFragment)
+        }
+    }
+
+    private fun onInitNav() {
+        bottomNavHomePage.setOnNavigationItemSelectedListener { it ->
+            when (it.itemId) {
+                R.id.itemCocktailsPage -> {
+                    viewPagerHomePage.currentItem = ItemBottomNav.COCKTAILS_ITEM.position
```
Comment (vvanan158) [inline app/src/main/java/com/sun/cooktails_app/ui/homepage/HomePageFragment.kt]: sử dụng ordinal

File: app/src/main/java/com/sun/cooktails_app/data/model/Cocktails.kt [hunk 4]
```diff
@@ -1,9 +1,9 @@
 package com.sun.cooktails_app.data.model
 
 data class Cocktails(
-        val id: Int,
-        val name: String,
-        val imageUrl: String
+    val id: Int,
+    val name: String,
+    val imageUrl: String
```
Comment (vvanan158) [inline app/src/main/java/com/sun/cooktails_app/data/model/Cocktails.kt]: set default hoặc nullable

File: app/src/main/java/com/sun/cooktails_app/utils/ItemBottomNav.kt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package com.sun.cooktails_app.utils
+
+enum class ItemBottomNav(var position: Int) {
+    COCKTAILS_ITEM(0),
+    FILTER_ITEM(1),
+    FAVORITE_ITEM(2)
+}
```
Comment (vvanan158) [inline app/src/main/java/com/sun/cooktails_app/utils/ItemBottomNav.kt]: sử dụng ordinal thì không cần truyền thêm position đâu bro

**Review by vvanan158 — APPROVED** [review]

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**